docs/troubleshoot/app-framework/appserver-error-codes.md

```diff
@@ -3243,68 +3243,68 @@ The following error message codes may appear on the app-server log. Use the foll
 
   **Reason:**
 
-  TODO
+  (_operatingSystemName_) is not a supported platform for _componentName_. Skipping _pluginid_... Supported: _requiredOperatingSystem_
 
   **Action:**
 
-  TODO
+  Contact the author of the component, or a system administrator.
 
 
 
 ### ZWED0154E":"(%s) is not a supported architecture for %s. Skipping (%s)... Supported: %s
 
   **Reason:**
 
-  TODO
+  (_operatingSystemName_) is not a supported platform for _componentName_. Skipping _pluginid_... Supported: _requiredOperatingSystem_
 
   **Action:**
 
-  TODO
+  Contact the author of the component, or a system administrator.
```

Comment:
Or check if you're running an outdated version of the plugin, or remove the plugin, or migrate zowe to the supported architecture.